tests/jobs/test_triggerer_job.py

```diff
@@ -586,3 +613,9 @@ def non_pytest_handlers(val):
     assert qh.__class__ == LocalQueueHandler
     assert qh.queue == listener.queue
     listener.stop()
+    listener = setup_queue_listener()
+    assert handler not in non_pytest_handlers(log.handlers)
+    qh = log.handlers[-1]
+    assert qh.__class__ == LocalQueueHandler
+    assert qh.queue == listener.queue
+    listener.stop()
```

Comment:
I suspect duplicating these lines wasn't intentional, yeah?

```suggestion
```

Reply:
Yep, thanks for pointing it out. I didn't notice it 🤦‍♂️